### DISCUSSION TITLE
Unable to create webhook for CollectionCreated event

### DISCUSSION THREAD

**Discussion by alta-atc** [saleor/saleor#11116] [Q&A]
Hi Gang,

Long time reader, first time caller. Big fan of the show.

We are building an integration to sync hundreds of thousands of products & related data from firestore into saleor and then tracking changes in firestore with webhooks. So far, it has been fairly smooth sailing.

We are having an issue creating a webhook for `CollectionCreated` event in Saleor. I am attaching the error. We are using this same method successfully for other webhooks... just this one has an issue. Appreciate any help that can be given.

![image (5)](https://user-images.githubusercontent.com/102269444/199291619-1f673cf4-dc43-4e02-a34a-2c0b3b3358f5.png)

```
mutation {
  webhookCreate(
    input: {
      app: "QXBwOjM="
      name: "collection_created_rowy"
      targetUrl: "https://afasedfarafgaew.a.run.app/wh/Hooks/6ygpyMrVO2Vuq5ytp3Ie"
      asyncEvents: [COLLECTION_CREATED]
      query: "subscription{event{...on CollectionCreated{collection{id name slug seoTitle seoDescription description }}}"
    }
  ) {
    errors {
      field
      message
    }
    webhook {
      id
      name
      subscriptionQuery
    }
  }
}
```

And the response 

```
{
  "data": {
    "webhookCreate": {
      "errors": [
        {
          "field": "query",
          "message": "Subscription query is not valid"
        }
      ],
      "webhook": null
    }
  },
  "extensions": {
    "cost": {
      "requestedQueryCost": 0,
      "maximumAvailable": 50000
    }
  }
}
```

**Comment by fowczarek** [comment]
Hi @alta-atc,
I think you miss one `}` at the end of `query` in variables. 

Bellow input should work. You need to replace only `URL` 
```
    input: {
      app: "QXBwOjM="
      name: "collection_created_rowy"
      targetUrl: "<URL_HERE4>"
      asyncEvents: [COLLECTION_CREATED]
      query: "subscription{event{...on CollectionCreated{collection{id name slug seoTitle seoDescription description }}}}"
    }
```

**Reply by alta-atc** [reply]
Thank you, I will give this a shot and report back
